torch/jit/__init__.py

```diff
@@ -462,6 +462,9 @@ def wrapper(func):
         executor_options = {'optimize': True}
         for name in executor_options:
             executor_options[name] = kwargs.pop(name, executor_options[name])
+        if len(kwargs) != 0:
+            raise TypeError("got unexpected keyword arguments: {}".format(",".join(kwargs.keys())))
```

Comment:
nit: it's better to use `', '.join(...)` (the space after the comma improves readability)